Week1/node.js

```diff
@@ -0,0 +1,184 @@
+  "INSERT INTO city (id, name, country_code, population) VALUES (2, 'Rotterdam','NLD', 623652)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (3, 'Kabul','KAB', 4600000)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (4, 'Bucharest','BUC', 1836000)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (5, 'Stockholm','STO',  965232)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (6, 'Damascus', 'DAM',1711000)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (7, 'Dodoma', 'DOD',2084000)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (8, 'Bern', 'BER', 133115)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (9,'Arnhem','NLD',138020)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (10,'Breda','NLD',160398)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (11,'Huambo','HUA',163100)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (12,'Lobito','LOB',130000)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (13,'Benguela','BEN',128300)",
+  "INSERT INTO city (id, name, country_code, population) VALUES (14,'Zaanstad', 'ZAA',135621)",
+];
+
+for (let i in insert_cities) {
+  // console.log('Going to run ', insert_cities[i]);
+  database.query(insert_cities[i], error => {
+    if (error) {
+      throw error;
+    }
+    console.log([i] + 'success');
+  });
+}
+
+// What are the names of the countries with population greater than 8 million
+database.query(`SELECT * FROM country WHERE Population > 8000000`, (error, result) => {
+  if (error) throw error;
+
+  console.log(`\n Countries with population more than 8 million:`);
+  result.forEach(country => {
+    console.log(`${country.Name} , ${country.Population}`);
+  });
+});
+
+// Names of the countries that have “land” in their names ?
+database.query(`SELECT * FROM country WHERE Name LIKE '%land%' `, (error, result) => {
+  if (error) throw error;
+  console.log(`\n Countries that have “land” in their names:`);
+  result.forEach(land => {
+    console.log(land.Name);
+  });
+});
+
+// Cities with population between 500,000 and 1 million ?
+database.query(
+  `SELECT * FROM city WHERE Population BETWEEN 500000 AND 1000000 `,
+  (error, result) => {
+    if (error) throw error;
+    console.log(`\n Cities with population between 500,000 and 1 million is :`);
+    result.forEach(city => {
+      console.log(city.Name);
+    });
+  },
+);
+
+// Countries in the Europe continent ?
+
+database.query(`SELECT * FROM country WHERE Continent = 'Europe' `, (error, result) => {
+  if (error) throw error;
+
+  console.log(`\n Countries in Europe continent:`);
+  result.forEach(country => {
+    console.log(country.Name);
+  });
+});
+
+// Countries in descending order, based on their surface areas.
+
+database.query(`SELECT * FROM country ORDER BY SurfaceArea DESC`, (error, result) => {
```

Comment:
I believe the column name is `surface_area`

Reply:
That is right , thanks